src/twisted/python/compat.py

```diff
@@ -822,6 +822,32 @@ def _bytesRepr(bytestring):
         return 'b' + repr(bytestring)
 
 
+
+def toBytes(s, encoding="ascii", errors="strict"):
+    """
+    Convert L{unicode} string to L{bytes}.
+
+    @param s: The string to convert.
+    @param encoding: The encoding to pass to L{str.encode}.
+    @param errors: The error handling scheme to pass to L{str.encode).
+
+    @raise TypeError: The input is not L{unicode} or L{bytes}.
+
+    @return: The encoded string.  If I{s} is L{bytes} or L{None}, just return
+             I{s}.
+    @rtype: L{bytes}
+    """
+    if s is None:
+        return s
+    elif isinstance(s, bytes):
+        return s
+    elif isinstance(s, unicode):
+        return s.encode(encoding=encoding, errors=errors)
+    else:
+        raise TypeError("Expected {} to be unicode or bytes, not: {}".format(
+            s, type(s)))
```

Comment:
There's lots of funny formatting like this throughout the diff.

Reply:
I don't understand your feedback.